packages/date-picker/src/panel/date.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <transition name="el-zoom-in-top" @after-enter="handleEnter" @after-leave="handleLeave">
+    <!-- Prevent mousedown from stealing focus from date input -->
     <div
+      @mousedown.prevent="noop"
```

Comment:
I trust that this fixes the problem, but to satisfy my curiosity, can you explain what this `@mousedown.prevent` accomplishes? I believe `.prevent` is `preventDefault`, right? So, does that mean that for anything in this container, when mousedown occurs, it prevents the default behavior (i.e. focusing)?

Reply:
Yes, you summarized it well. I learned this trick from some SO answer and it has been insanely useful to keep the right thing focused.